app/lib/methods/serializeAsciiUrl.test.ts

```diff
@@ -0,0 +1,30 @@
+import { serializeAsciiUrl } from '.';
+
+const ASCIIUrl = 'https://чат24.рф';
+const NonASCIIUrl = 'open.rocket.chat';
+const ASCIIUrlSerialized = 'https://xn--24-6kc6exa.xn--p1ai/';
```

Comment:
I'm wondering about the last slash. I guess that we need to implement a regex to remove the last slash, because we are expecting that the url doesn't have that slash.

Reply:
Why should we remove it?